src/developer/web-api/tracker.md

```diff
@@ -1994,6 +1997,20 @@ F8yKM85NbxW,Zy2SEgA61ys,2019-08-21T11:25:38.022Z,2019-03-19T00:12:16.624Z,2019-0
 F8yKM85NbxW,Zy2SEgA61ys,2019-08-21T11:25:38.022Z,2019-03-19T00:12:16.624Z,2019-08-21T11:31:33.410Z,2019-03-19T00:12:16.624Z,DiszpKrYNg8,false,false,false,"POINT (-11.7896 8.2593)",8.2593,-11.7896,,,,2019-08-21T11:25:38.388Z,2019-08-21T11:25:38.388Z,aW66s2QSosT,"Last Name",Johnson,TEXT
 ```
 
+##### Tracked Entities Collection limits { #webapi_tracker_entity_requests_limit }
+
+There are several ways of limiting the number of results a user is able to retrieve from the collection endpoint:
```

Comment:
These three limits have their own explanations (where the parameters are explained "very shortly") We need to update those areas with all of these. 
We could still create a new section, but refer to those parameters here instead of explaining everything here?

Reply:
@ameenhere I just linked the system setting to my explanation. 
Same goes for the program limits. 
However, I can't find the TET limits explained anywhere else, did you see it somewhere?

Reply:
https://docs.dhis2.org/en/use/user-guides/dhis-core-version-master/configuring-the-system/programs.html#create_tracked_entity